wrs/ceph.sh

```diff
@@ -42,6 +42,12 @@ stop ()
 {
     if [[ "$nodetype" == "controller" ]] || [[ "$nodetype" == "storage" ]]
     then
+        if [[ "$system_type" == "All-in-one" ]] && [[ "$system_mode" == "simplex" ]]
```

Comment:
Why do we leave ceph service to run in all-in-one and simplex mode?

Reply:
This is needed for the story that is currently in progress: https://storyboard.openstack.org/#!/story/2002844 and this review https://review.openstack.org/#/c/611627/

Short story: On 1 node configuration we need Ceph to be operational when node is locked otherwise sysinv will give errors when trying to configure it (the ceph monitor has to continue running otherwise cluster is unresponsive). In multinode configurations this is not a problem as we are not able to lock the active controller.

Reply:
Some more details: This file (ceph.sh) is not a systemd/init script but is used by MTC to stop services on host lock (mtc executes all files in /stc/services.d/<node personality/* with the 'stop' argument - it is then the responsibility of the scripts to decide if & how it stops each individual service).
Ceph lifetime is actually managed by pmon (started, monitored & restarted in case of errors) mtc only trigger agracefull shutdown of ceph on unlock.

Reply:
@oponcea Thanks for your detailed explanation. No objection about this change.